### PR TITLE
** MERGE ME PLEASE ** Datatrawler fixup

### PR DESCRIPTION
**Pull request by anguss00**
All good to merge

### CODE REVIEW

**Comment by pmbohm** [comment]
@anguss00 OMG what happened here?

**Comment by anguss00** [comment]
@pmbohm cleaned up and ready for a look over now

File: src/test/javascript/portal/cart/DataTrawlerDownloadHandlerSpec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            };
+            testHandlerParams = {
+                emailAddress: 'bob@example.com'
+            };
+        });
+
+        it('builds the correct URL', function() {
+            url = clickHandler(testCollection, testHandlerParams);
+
+            expect(url).toStartWith(handler.getAsyncDownloadUrl('datatrawler'));
```
Comment (anguss00) [inline src/test/javascript/portal/cart/DataTrawlerDownloadHandlerSpec.js]: @pmbohm I probably could do with some more test coverage here!

File: grails-app/conf/Config.groovy [hunk 2]
```diff
@@ -206,12 +206,18 @@ knownServers = [
     [
         uri: 'https://www.cmar.csiro.au/geoserver/wms',
```
[the remark this answers is not in the document]
Comment (anguss00) [reply grails-app/conf/Config.groovy]: Not necessarily - this is just changed to http here as the endpoint changed

File: grails-app/services/au/org/emii/portal/DataTrawlerService.groovy [hunk 3]
```diff
@@ -1,25 +1,27 @@
...
-        return params.request.toString()
+        return [ url: _getJobReportUrl() ] as JSON
     }
 
-    def onResponseSuccess = { resp, xmlReader ->
-        log.info("POST response status: ${resp.statusLine}")
-        return new StreamingMarkupBuilder().bindNode(xmlReader).toString()
+    def _getJobReportUrl() {
+        // TODO: replace placeholder with status url
+        return grailsApplication.config.csiro.url
```
[the remark this answers is not in the document]
Comment (anguss00) [reply grails-app/services/au/org/emii/portal/DataTrawlerService.groovy]: Nope, I put it as a placeholder as I assume that there will be a status URL at some point

File: web-app/js/portal/lang/en.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     downloadAsCsvLabel: 'CSV',
     downloadAsSubsettedNetCdfLabel: 'NetCDF',
     downloadAsPointTimeSeriesCsvLabel: 'CSV',
-    downloadAsDataTrawlerCsvLabel: 'CSV - DataTrawler',
+    downloadAsDataTrawlerCsvLabel: ' CSIRO Data Trawler extracted data',
```
[the remark this answers is not in the document]
Comment (anguss00) [reply web-app/js/portal/lang/en.js]: Yup, this is the string that was requested (minus the whitespace). Can throw it back if required

**Comment by anguss00** [comment]
@pmbohm changes up

File: src/test/javascript/portal/cart/DataTrawlerDownloadHandlerSpec.js (hunk 1 above)
Comment (anguss00) [reply src/test/javascript/portal/cart/DataTrawlerDownloadHandlerSpec.js]: Added more coverage, thanks for the tip @anguss00. 
No problems, @anguss00!

**Comment by pmbohm** [comment]
@anguss00 Rebased and looking good. Merge while the irons hot!

**Pull request merged**